src/js/disability-benefits/526EZ/components/IntroductionPage.jsx

```diff
@@ -1,73 +1,132 @@
+                    <p>We assign disability ratings with a severity from 0% to 100% for each disability claim. This rating can change if there are changes in your condition. We’ll decide a claim for increase based on the medical evidence and supporting documents you turn in with your claim.</p>
+                    <p><a href="/disability-benefits/eligibility/ratings/">Learn how VA assigns disability ratings</a>.</p>
+                  </div>
+                </div>
+                <br/>
+              </div>
             </li>
             <li className="process-step list-two">
               <div><h5>Apply</h5></div>
-              <p>Complete this education benefits form.</p>
+              <p>Complete this disability compensation benefits form.</p>
               <p>After submitting the form, you’ll get a confirmation message. You can print this for your records.</p>
             </li>
             <li className="process-step list-three">
               <div><h5>VA Review</h5></div>
-              <p>We usually process claims within 30 days. We’ll let you know by mail if we need more information.</p>
-              <p>We offer tools and counseling programs to help you make the most of your educational options. <a href="/education/tools-programs/">Learn about career counseling options.</a></p>
+              <p>We process applications in the order we receive them. We may contact you if we have questions or need more information.</p>
             </li>
             <li className="process-step list-four">
               <div><h5>Decision</h5></div>
-              <p>You’ll get a Certificate of Eligibility (COE), or award letter, in the mail if we've approved your application. Bring this to the VA certifying official at your school.</p>
-              <p>If your application wasn’t approved, you’ll get a denial letter in the mail.</p>
+              <p>Once we’ve processed your claim, you’ll get a notice in the mail with our decision.</p>
             </li>
           </ol>
         </div>
-        <SaveInProgressIntro
-          buttonOnly
-          messages={this.props.route.formConfig.savedFormMessages}
-          pageList={this.props.route.pageList}
-          startText="Start the Education Application"
-          {...this.props.saveInProgressActions}
-          {...this.props.saveInProgress}/>
+        {!user.login.currentlyLoggedIn && <a className="usa-button-primary" href="/disability-benefits/526/apply-for-increase/introduction/" onClick={this.authenticate}>Sign In and Verify Your Identity</a>}
+        {user.login.currentlyLoggedIn && <RequiredLoginView
+          verify
+          serviceRequired={['disability-benefits']}
+          user={user}
+          loginUrl={loginUrl}
+          verifyUrl={verifyUrl}>
+          <SaveInProgressIntro
+            {...this.props}
+            buttonOnly
+            handleLoadPrefill={this.handleLoadPrefill}/>
+        </RequiredLoginView>}
```

Comment:
Can we DRY this up? It looks really similar to the block above.

Reply:
Sounds good, I will move all of this into a FormStartControls component.